### ISSUE TITLE
Throwing error when progress is 0?

### ISSUE THREAD

**Issue by ksloan** [react-component/progress#35]
```js
import { Circle } from 'rc-progress';

ReactDOM.render(<div>
  <Circle percent={0} strokeWidth="4" strokeColor="#D3D3D3" />
</div>, container);
```

is throwing 

```
Uncaught TypeError: Cannot read property 'style' of null
    at Progress.componentDidUpdate (enhancer.js:42)
```

complaining about this line https://github.com/react-component/progress/blob/85ab19473167e349dc8f1b6368a4d85b6500d6db/src/enhancer.js#L4

**Comment by paranoidjk** [comment]
@benjycui 你先处理下，估计是昨天那个pr的影响，我还没到

**Comment by benjycui** [comment]
@paranoidjk fixed, need a patch.

I am not npm owner of rc-progress.

**Comment by paranoidjk** [comment]
@ksloan Published `rc-progress@2.2.1`, please upgrade your deps.
